azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/manager/AvatarViewManager.kt

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.presentation.manager
+
+import com.azure.android.communication.ui.configuration.CommunicationUILocalDataOptions
+
+internal class AvatarViewManager(private val communicationUILocalDataOptions: CommunicationUILocalDataOptions?) {
```

Comment:
Any reason why we cannot make this public val?  Then the function below is not needed